keps/sig-node/2902-cpumanager-distribute-cpus-policy-option/README.md

```diff
@@ -116,6 +123,24 @@ NOTE: The striping operation after all CPUs have been evenly distributed will be
 
 We will extend both the unit test suite and the E2E test suite to cover the new policy option described in this KEP.
 
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+##### Unit tests
+
+- `k8s.io/kubernetes/pkg/kubelet/cm/cpumanager`: `20250205` - 85.5% of statements
+
+##### Integration tests
+
+Not Applicable as Kubelet features don't have integration tests.
+
+##### e2e tests
+
+Currently no e2e tests are present for this particular policy option. E2E tests will be added as part of Beta graduation.
```

Comment:
Maybe you can describe the tests you want to add at least.

Reply:
Sure, will add more details in the KEP.